### DISCUSSION TITLE
Allow ceramic tiles as an option for road surfaces

### DISCUSSION THREAD

**Discussion by latenightdef** [streetcomplete/StreetComplete#5950] [Feedback]
In my area, some outdoor corridors and sidewalks are using ceramic tiles for the surface. It'd be great if I can tag the surface as  `surface=tiles` right from StreetComplete app.

Here's an example of my university's outdoor corridor:

![20241005_184717](https://github.com/user-attachments/assets/2eb9e379-6aa9-4031-86a5-8ebf6e37c550)

**Comment by mnalis** [comment]
Note that [surface=tiles](https://wiki.openstreetmap.org/wiki/Tag:surface%3Dtiles) has relatively few uses, and is much more likely (about [114k vs 1.5k](https://taginfo.openstreetmap.org/tags/surface%3Dpaving_stones#combinations))  to be tagged as [surface=paving_stones](https://wiki.openstreetmap.org/wiki/Tag:surface%3Dpaving_stones) with [surface=excellent](https://wiki.openstreetmap.org/wiki/Tag:smoothness%3Dexcellent), see e.g. https://wiki.openstreetmap.org/wiki/File:Pretty_natural_stone_used_for_paving.jpg

StreetComplete contains quests both for `surface` and for `smoothness` (_"surface quality"_), although the latter needs to be enabled manually in settings), if that is good enough for your use case.

There are also [several tags to specify more details](https://wiki.openstreetmap.org/wiki/Tag:surface%3Dpaving_stones#Subtags) about paving stones (e.g. [paving_stones:material](https://taginfo.openstreetmap.org/keys/paving_stones%3Amaterial#wiki)  might be of particular interest), but StreetComplete does not ask for those, as such level of detail has low ratio of effort/usefulness.
